### DISCUSSION TITLE
Behringer X-Touch Mini - Profiles and Overlay Template

### DISCUSSION THREAD

**Discussion by onkytonk** [rsjaffe/MIDI2LR#538] [Show and tell]
### Behringer X-Touch Mini - Custom Layout - By Joseph Naim

Hey all, 
I have finally got around to updating my very popular custom layout for the Behringer X-Touch Mini!
The new version is updated to work with the current latest versions of LR Classic (Currently 12.x) and MIDI2LR (Currently 5.x).

I think its one of the best layouts going around, but love to hear feedback! 

Included:
- Installation instructions
- X-Touch Editor Presets
- MIDI2LR Profiles
- MIDI2LR Lightroom config
- Overlay Template (PSD Format)

Version 3.00
**Link: https://drive.google.com/uc?export=download&id=1ETpBydF9yPbNrgkYw7VU5eQ0otGsZAkH**

<a href="https://www.buymeacoffee.com/josephnaimau" target="_blank"><img src="https://cdn.buymeacoffee.com/buttons/default-orange.png" alt="Buy Me A Coffee" height="41" width="174"></a>

### Layer A
Encoders: Exposure – Contrast – Highlights – Shadows – Whites – Blacks – Temperature – Tint 
(Pressing any encoder button resets that particular setting)
Top Buttons: Select Active – Match Exposure – Sync Settings – Virtual Copy – Reset All – Convert to B&W – Select Left – Select Right
Bottom Buttons: Copy Settings – Paste Settings – Paste Previous – Before/After – Undo – Redo – Previous – Next
Fader: Exposure
### Layer B
Encoders: Texture – Clarity – Dehaze – Vibrance – Saturation – Crop – Rotation – Zoom
(Pressing any encoder button resets that particular setting)
Top Buttons: Default Profile – Detail Profile – Hue Profile – Saturation Profile – Luminance Profile – B&W Profile – Lights Out – Full Screen
Bottom Buttons: Decrease Rating – Increase Rating – Reject Flag – Toggle Flag – Info – Auto Tone – Previous – Next
Fader: Exposure
### How to use Profiles
To change to a different profile, choose Layer B and then select from:
Default – Detail – Hue – Saturation – Luminance – B&W
Once a profile is selected, stay in Layer B and the encoders will now control the following:
Detail: Sharpening Amount – Sharpening Radius – Sharpening Detail – Sharpening Masking – NR Luminance – NR Luminance Detail – NR Color – NR Color Detail
Hue/Saturation/Luminance/B&W: Red – Orange – Yellow – Green – Aqua – Blue – Purple – Magenta
### How to use Match Exposure/Sync Settings
Match Exposure: When you have selected 2 or more photos, it matches the overall exposure (not the setting) based on the first photo that was selected.
Sync Settings: When you have selected 2 or more photos, it synchronises all settings based on the first photo that was selected.
Select Left/Select Right: Select multiple photo to the left or right respectively.
Select Active: Unselects multiple photos, and returns to only the active photo.
1) Start from the main photo you have edited, then Select Left or Select Right to select all the photos you want to affect.
2) Press either Match Exposure or Sync Settings (depending on what you would like to do). All the photos selected will be affected.
3) Press Select Active to unselect and return to only the active photo.
### Extra Notes
All other buttons function the same in all profiles.
Fader always controls Exposure

![Behringer Template v3 0](https://github.com/rsjaffe/MIDI2LR/assets/29836445/e1054afb-26b8-477e-bf47-67622c393616)

**Comment by skrasii** [comment]
Hi @onkytonk, thank you for your work. Could you please update the G Drive link to Behringer X-touch Mini profiles? Current link one doesn't work anymore :( Thanks

**Comment by agrenott** [comment]
I had a copy of the original zip file, here it is (I guess the original author won't mind):
[Midi2LR for Behringer X-Touch Mini Package 2.03.zip](https://github.com/rsjaffe/MIDI2LR/files/12358621/Midi2LR.for.Behringer.X-Touch.Mini.Package.2.03.zip)

**Comment by onkytonk** [comment]
OP updated with new Version 3.00. 😄

**Comment by hdstead** [comment]
Hi, thanks for this.  I have not been able to download the files from 
Link: https://drive.google.com/file/d/1ETpBydF9yPbNrgkYw7VU5eQ0otGsZAkH/view?usp=sharing .  It opens, showing the individual files, but from there I can't seem to find a way to download the zip file.  Any suggestion would be much appreciated.

**Comment by hdstead** [comment]
I am having a problem download the zip file Midi2LR for Behringer X-Touch Mini Package v3.00.rar.  What I see from the link is a screen that asked me that show the content of the zar file, asking who I would like to open the files inside.  My zip utility tries, but comes back saying the file location is not there.  I also don't see a download button, and my Firefox download, doesn't do the trick.  My apologies if I am missing something here.  Thanks in advance.

**Reply by onkytonk** [reply]
When I click the link in a web browser on a desktop, there's a download icon in the top right corner. It comes up for me regardless of whether or not I'm signed in to Google Drive. 

Can you not see this?

![Screenshot 2023-09-10 072424](https://github.com/rsjaffe/MIDI2LR/assets/29836445/f3ff0de9-c1f2-40c2-9766-dd31f4a763c7)

**Reply by onkytonk** [reply]
I've changed the link to a Direct Download - it should work for you now. 😄

**Reply by hdstead** [reply]
Thank you very much.  Worked great!  One click and the download arrived instantly.

**Comment by hdstead** [comment]
Yes, the download button, gave an error message.  It is a history to me.
Is there another repository, I can try?

**Reply by onkytonk** [reply]
Did the updated direct download link not work?

**Comment by hdstead** [comment]
Great, simplicity, one click and file instantly arrived in my downloads.
Thanks again, Sir.

**Comment by lkurasinski** [comment]
Hi @onkytonk, I appreciate your time for sharing this;)
Do you know is there a way to use it on MacOS? At the moment X-Touch Editor is not available for apple and it seems that loading your layer presets is a necessary step to make your profile work. Have you heard about any workaround?

**Reply by onkytonk** [reply]
Oh I had no idea it wasn’t available for Mac. How annoying?! 😅

Unfortunately you would need a virtual machine, or find a friend with a PC. Once the presets are installed on the device, you won’t ever need that step again. 

I’ve also read about “Wine” software for the Mac that allows you to run windows applications within Mac OS. WineHQ and wineskinserver are a couple of examples.

**Reply by lkurasinski** [reply]
Thanks for the hint, I've used another windows machine👍
~~One more thing @onkytonk, have you figure out how to apply local adjustments? From what I found out so far,  toa chive that I would need another dedicated profile, with separate mapping? By local adjustments I mean e.g. modify exposure in masked area~~

I've finally managed to map keys as I wanted to, including localAdjustments ^^

**Comment by djsocrate** [comment]
I'm having trouble since when i switch to the next photo, the levels of the knobs is the one of the previous photo and not the current one so it fail to sync. I've repeated  all the steps but nothing change. I'm about to loose it. Any idea? Thanks!

**Reply by onkytonk** [reply]
I haven’t encountered that issue before, so I’m not sure where to start. 

Maybe try a full hardware reset on the Xtouch, and redo all the steps meticulously?

Hard reset (factory reset) the Xtouch mini. Which is done by:
a. Unplug it
b. Press both And B buttons together, and while pressing - plug it in.
c. Wait for few secs for A and B lights to slowly flicker.
d. When only A stays lit - the hard reset is completed and you can let go. The moment you let go, 1 tiny hair light of each of the upper row rotating buttons will light up as well.

**Reply by djsocrate** [reply]
Solved!! Thanks! This was the right procedure! Thanks a lot!

**Reply by unimatrixZxero** [reply]
Awesome this reset procedure fixed my flickering nobs issue. I had to redo all my bindings but now they work smoothly.
